include/aspect/velocity_boundary_conditions/gplates.h

```diff
@@ -38,80 +40,47 @@ namespace aspect
     {
       /**
        * GPlatesLookup handles all kinds of tasks around looking up a certain
-       * velocity boundary condition from a gplates .gpml file. This class
-       * keeps around the contents of two sets of files, corresponding to two
-       * instances in time where GPlates provides us with data; the boundary
-       * values at one particular time are interpolated between the two
-       * currently loaded data sets.
+       * velocity boundary condition from a gplates .gpml file.
```

Comment:
Is the previous description no longer the case? How do you do interpolation in time now?